examples/httpd/convox.yml

```diff
@@ -2,3 +2,11 @@ services:
   web:
     image: httpd
     port: 80
+  worker:
+    build: ./worker
+    scale: 0
+timers:
+  cleanup:
```

Comment:
can you rename the timer to `example` or something more descriptive?
`cleanup` can be misleading.